tests/5_dobby_container_runtime_test.sh

```diff
@@ -18,555 +18,545 @@
 
-        local testid="5.21"
-        local desc="Ensure that seccomp profile is enabled for the container"
-        local check="$testid - $desc"
-        local output
-
-        output=$(grep Seccomp /proc/$Container_PID/status | awk '{print $2}')
-
-        if [ "$output" == "0" ]; then
-        	fail "$check"
-		verbosetxt "Seccomp is not enabled"
-		return
-	elif [ "$output" == "1" -o "$output" == "2" ]; then
-		pass "$check"
-	else
-		fail "$check"
-		verbosetxt "Seccomp is not enabled"
-	fi
+    output=$(grep Seccomp /proc/$Container_PID/status | awk '{print $2}')
 
+    if [ "$output" == "0" ]; then
+        fail "$check"
+        verbosetxt "Seccomp is not enabled"
+        return
+    elif [ "$output" == "1" -o "$output" == "2" ]; then
+        pass "$check"
+    else
+        fail "$check"
+        verbosetxt "Seccomp is not enabled"
+    fi
 }
-test_5_24() {
 
-	local testid="5.24"
-        local desc="Ensure that cgroup is defined for the container"
-        local check="$testid - $desc"
-	local output
-
-	output=$(cat /proc/$Container_PID/cgroup)
-	while read line;
-        do
-		line=$(echo $line | grep -o ':/[^"]*' | cut -d '/' -f2-)
-                if [ "$line" != "$containername" ]; then
-                        fail "$check"
-                        return
-                fi
-        done <<< "$output"
-
-        pass "$check"
-	
+test_5_24() {
+    local testid="5.24"
+    local desc="Ensure that cgroup is defined for the container"
+    local check="$testid - $desc"
+    local output
+
+    output=$(cat /proc/$Container_PID/cgroup)
+    while read line;
+    do
+        line=$(echo $line | grep -o ':/[^"]*' | cut -d '/' -f2-)
+        if [ "$line" != "$containername" ]; then
+            fail "$check"
+            return
+        fi
+    done <<< "$output"
 
+    pass "$check"	
 }
 
 test_5_24_1() {
-
-        local testid="5.24.1"
-        local desc="Ensure that GPU cgroup restrictions are enabled in supported platforms (Only supported for Mali platforms now)"
-        local check="$testid - $desc"
-        local output
-	local FILE
+    local testid="5.24.1"
+    local desc="Ensure that GPU cgroup restrictions are enabled in supported platforms (Only supported for Mali platforms now)"
+    local check="$testid - $desc"
+    local output
+    local file
 	
-
-	FILE="/sys/fs/cgroup/gpu/$containername/gpu.limit_in_bytes"
-	if [ -f $FILE ]; then
-		output=$(cat /sys/fs/cgroup/gpu/$containername/gpu.limit_in_bytes)
-        	total=$(cat /sys/fs/cgroup/gpu/gpu.limit_in_bytes)
-
-        	if [ "$output" == "0" -o "$output" == "-1" -o "$output" == "$total" ]; then
-          		fail "$check"
-         		return
-        	fi
-          	pass "$check"
-	else
-		warn "$check"
-		verbosetxt "${bldcynclr} GPU cgroup is not supported in this platform"
-	fi
-
+    file="/sys/fs/cgroup/gpu/$containername/gpu.limit_in_bytes"
+    if [ -f $file ]; then
+        output=$(cat /sys/fs/cgroup/gpu/$containername/gpu.limit_in_bytes)
```

Comment:
If you are refactoring `file` name, you can use it here too (look on test_5_11 on how it should be done), this can be done to other occurrences.